### DISCUSSION TITLE
Is there a way to include custom variables to webhook payload?

### DISCUSSION THREAD

**Discussion by alex-noel** [strapi/strapi#6246] [Legacy Topics]
I'm trying to trigger a custom pipeline in CircleCI via Strapi webhooks - https://circleci.com/docs/api/v2/#trigger-a-new-pipeline. In order for it to be triggered I need to pass two variables into body - branch name, and parameters.
Is there any way to extend the payload of a webhook to include custom data? Can I somehow override the core functionality the same way as with plugins?

**Comment by lauriejim** [comment]
For now you can only add custom headers, you can't customize the data you send in the request.

**Reply by divyansh0777** [reply]
@lauriejim Can you please tell me how can I retrieve content-data from the webhook API?

**Reply by lauriejim** [reply]
Hello! Can you please precise your question.
I'm not sure to understand.
